modules/ROOT/pages/index.adoc

```diff
@@ -4,7 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-API Community Manager (ACM) overview here.
+MuleSoft enables your company to build APIs, catalog them with Exchange, and present them to software developers inside and outside your company with API Community Manager (ACM).
```

Comment:
suggestion only (take or leave)

API Community Manager (ACM) enables your company to build APIs, catalog them with Anypoint Exchange, and make them available to software developers inside and outside of your company.

Reply:
Thank you. I like that better.